Automattic/wp-calypso
# WooCommerce: Connect store address fields to state, persist to server

See #14775 

To test, go to
- http://calypso.localhost:3000/store/{site}
- Make sure the name, address, address2, city, state, postcode and country fields are displayed.
- Note that they are editable. Make some changes.
- Make sure the Next button works
- After you click the Next button, verify you are taken to the "egg" view (i.e. the list of tasks and checkboxes)
- Note that after you've done "Next" that the set_store_address_during_initial_setup flag will be set on WPCOM for that site.
- To get back to the address view, you can “reset” the site using the developer console at  https://developer.wordpress.com/docs/api/console/ -- you’ll need to POST to /sites/siteID/calypso-preferences/woocommerce setting set_store_address_during_initial_setup to 0
- Make sure the changes you made are there
- Lastly, run npm run test-client client/extensions/woocommerce/state and make sure everything passes


Caveats:
- I'm not happy with the userBeganEditing flag in PreSetupView but without it, you can start editing and then lose your edits on a later prop wash.  Open to advice on how to resolve that.
- Site Title is not editable yet
